nanoAOD/plugins/HadTruthProducer.h

```diff
@@ -68,6 +70,7 @@ class HadTruthProducer : public edm::stream::EDProducer<>
   edm::EDGetTokenT<edm::View<reco::GenParticle> > genLabel_;
   edm::EDGetTokenT<TrackingVertexCollection> trackingVertexLabel_;
   edm::EDGetTokenT<TrackingParticleCollection> trackingParticleLabel_;
+  
```

Comment:
Could you remove the newline here, the space after private (my OCD), and then put the header include in the cc file. Its better to keep the .h file clean of includes if they're not explicitly necessary for the class interface. Thanks!